cpp/src/io/parquet/page_data.cu

```diff
@@ -270,7 +270,7 @@ __device__ void gpuDecodeStream(
     }
     if (s->error) { break; }
 
-    batch_len = min(num_input_values - value_count, 32);
+    batch_len = min(num_input_values - value_count, BATCH_SZ); //defined in parquet_common.h
```

Comment:
`BATCH_SZ` can't be found in https://github.com/goodengineer/cudf/blob/patch-4/cpp/src/io/parquet/parquet_common.h